src/devices/Ccs811/MeasurementArgs.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using UnitsNet;
+
+namespace Iot.Device.Ccs811
+{
+    /// <summary>
+    /// Arguments of the Measurement Threshold event
+    /// Contains the measurements done and potential error
+    /// </summary>
+    public class MeasurementArgs
+    {
+        /// <summary>
+        /// True if measurement is successful
+        /// </summary>
+        public bool MeasurementSuccess { get; set; }
+
+        /// <summary>
+        /// Equivalent CO2 in ppm
```

Comment:
comments still shows unit but now it is irrelevant since user can pick the unit, you can change it to perhaps suggested unit so that user knows what to display, i.e.: `Equivalent CO2. Sensor internally measures [ppm - expand name].`